src/css/base.scss

```diff
@@ -3,26 +3,59 @@ $primary-background: radial-gradient(circle, rgba(63,94,251,1) 0%, rgba(252,70,1
 ;
 
 body {
-
+ margin: 0;
+ font-family: "RocknRoll One"
 }
 
 header {
   display: flex;
   justify-content: space-between;
+  background-color: #BAD4FA;
+  width: 100%;
+  .user {
+    margin-right: 10px;
+    margin-top: 30px;
+  }
+  .title {
+    margin-left: 10px;
+  }
 }
 
-.profile {
-  text-align: center;
+h2 {
+  margin-left: 10px;
 }
 
+h3 {
+  background-color: #4594EB;
```

Comment:
Think about adding variables for colors and dropping them in wherever you use a color code.